serverless/index-serverless-project-settings.asciidoc

```diff
@@ -25,3 +27,16 @@ include::./pages/transforms.asciidoc[leveloffset=+3]
 include::./pages/integrations.asciidoc[leveloffset=+2]
 
 include::./pages/fleet-and-elastic-agent.asciidoc[leveloffset=+2]
+
+include::./pages/manage-org.asciidoc[leveloffset=+2]
```

Comment:
I'd either remove this page OR rename it to "manage users and roles" and keep only what's relevant to serverless. For the rest, we can link out to the cloud docs

The concept of "managing the org" doesn't fit under managing projects